Composer/packages/server/jest.config.js

```diff
@@ -4,5 +4,5 @@ const { createConfig } = require('@bfc/test-utils');
 
 module.exports = createConfig('server', 'node', {
   setupFiles: [path.resolve(__dirname, 'src/__tests__/setupEnv.ts')],
-  testPathIgnorePatterns: ['src/__tests__/setupEnv.ts'],
+  testPathIgnorePatterns: ['src/__tests__/setupEnv.ts', 'orchestratorBuilder'],
```

Comment:
I've added this UT to the ignore list for now since it does a model download at the start of all tests (180MB+) which is needed for other tests. I am aware is a codesmell, any suggestions appreciated. 

Orchestrator team have mentioned there is a smaller model that can be made available for UTs in the future.